Python/bigtable/write_bigtable.py

```diff
@@ -0,0 +1,83 @@
+import logging
```

Comment:
@Amar3tto  Also please update README.md file. We might also add some comments there regarding examples that require some objects created in the project (like big query instances). Or we will have to add README into the example folder.